content/storage-history/history.textile

```diff
@@ -421,35 +423,27 @@ Retrieve "presence":/presence-occupancy/presence history using the "@history()@"
 The following example retrieves a paginated list of historical presence events published:
 
 ```[realtime_javascript]
-channel.attach(function() {
-  var presence = channel.presence;
-  presence.history({}, function(err, resultPage) {
-    if(err) {
-      console.log('Unable to get presence history; err = ' + err.message);
-    } else {
-      console.log(resultPage.items.length + ' presence events received in first page');
-      if(resultPage.hasNext()) {
-        resultPage.next(function(err, nextPage) { ... });
-      }
-    });
-  };
-});
+await channel.presence.enter('enter');
+await channel.attach();
+const history = await channel.presence.history();
+console.log(history.items.length + ' presence events received in first page');
+
+if (history.hasNext()) {
+  const nextHistory = await history.next();
+  console.log(nextHistory.items.length);
+}
 ```
 
 ```[realtime_nodejs]
-channel.attach(function() {
-  var presence = channel.presence;
-  presence.history({}, function(err, resultPage) {
-    if(err) {
-      console.log('Unable to get presence history; err = ' + err.message);
-    } else {
-      console.log(resultPage.items.length + ' presence events received in first page');
-      if(resultPage.hasNext()) {
-        resultPage.next(function(err, nextPage) { ... });
-      }
-    });
-  };
-});
+await channel.presence.enter('enter');
+await channel.attach();
```

Comment:
Attaching explicitly won't do anything here because you'll be implicitly attached by calling `channel.presence.enter`

Reply:
https://github.com/ably/docs/pull/2119/commits/6057f3d7eeee3bc2df3e016700f8e5af8561d633

There were a few references to this. I think I just had left them from existing code samples.